### ISSUE TITLE
`training_data` can not be a pandas data frame

### ISSUE THREAD

**Issue by JCZuurmond** [Azure/MachineLearningNotebooks#1665]
I see the following exception when  pandas data frame is passed as `training_data` in the [AutoMLConfig](https://docs.microsoft.com/en-us/python/api/azureml-train-automl-client/azureml.train.automl.automlconfig(class)?view=azure-ml-py#feedback).

``` text
ConfigException: ConfigException: Message: Input of type '<class 'pandas.core.frame.DataFrame'>' is not supported. Supported types: [azureml.data.tabular_dataset.TabularDataset]Please refer to documentation for converting to Supported types: https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.dataset.dataset?view=azure-ml-py InnerException: None ErrorResponse { "error": { "code": "UserError", "message": "Input of type '<class 'pandas.core.frame.DataFrame'>' is not supported. Supported types: [azureml.data.tabular_dataset.TabularDataset]Please refer to documentation for converting to Supported types: https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.dataset.dataset?view=azure-ml-py", "details_uri": "https://aka.ms/AutoMLConfig", "target": "training_data", "inner_error": { "code": "BadArgument", "inner_error": { "code": "ArgumentInvalid", "inner_error": { "code": "InvalidInputDatatype" } } } } }
```

The documentation says that `training_data` can be a pandas data frame.

Versions:
- Python 3.6
- SDK version: 1.36.0

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: be87853c-7d77-ec34-2bd2-a2f1e7104a3e
* Version Independent ID: 9c3f1114-c842-d0dd-ee02-7fae9974a545
* Content: [azureml.train.automl.AutoMLConfig class - Azure Machine Learning Python](https://docs.microsoft.com/en-us/python/api/azureml-train-automl-client/azureml.train.automl.automlconfig(class)?view=azure-ml-py)
* Content Source: [AzureML-Docset/stable/docs-ref-autogen/azureml-train-automl-client/azureml.train.automl.AutoMLConfig(class).yml](https://github.com/MicrosoftDocs/MachineLearning-Python-pr/blob/live/AzureML-Docset/stable/docs-ref-autogen/azureml-train-automl-client/azureml.train.automl.AutoMLConfig(class).yml)
* Service: **machine-learning**
* Sub-service: **core**
* GitHub Login: @DebFro
* Microsoft Alias: **debfro**

**Comment by slsu0424** [comment]
Hello - I have been running into the exact same issue as @JCZuurmond.  Is there any update on this?

**Comment by JCZuurmond** [comment]
Nope, no response on this PR yet. I don't remember what I did with this issue

**Comment by majdoulinamakbal** [comment]
Any update ? running into same problem ..

**Comment by majdoulinamakbal** [comment]
Hello, so here's how i solved the issue:

1. Register your pandas dataframe as a tabular dataset using Dataset.Tabular.register_pandas_dataframe(your_dataframe,your_datastore,'a description')
2. use the registered tabular dataset in your automl config

**Comment by kryukovaeks** [comment]
Thank you!

>
